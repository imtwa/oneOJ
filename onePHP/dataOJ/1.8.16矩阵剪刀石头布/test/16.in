77 13 28
RSPRPRPSPRPPS
PSRPRSRPSSRSS
RSRRSRRRSSSSS
PPRRPSPRPRSRR
SRRPPSRSSSPPR
PRSPPSPRRPPPP
SSRPSSRRPPRSR
SPRSSSPPSRRPR
PPRPSPPRSRPPR
PPRSSRPPPPRPP
RSSRPRSSPSRPP
RSPPRSPPRRSRR
PPPPRRPRSPRSS
PRSSSSRPRPPPP
RPRPRRSRRRPPP
PSSRPPRPSRPRS
RRPPSRPSRPSPR
PPRRSPSRRSPPR
PRRRPRRPSSRRR
SPPSSSSPRSRPP
RSRSSSPSRSRPR
SPRSRRRSRPRRS
PRPRSRSRRPPSR
RPSPPRRPSRSPR
PPRSRSRSSSPPS
PSSRPPSPRSPRR
SRSPSSPRRSRSP
SSRSPSSPPRSSR
SPSPRSSPSRPSR
PPRRPPPPRPSSP
RSPSRRSRSRRRR
SPPPRPSSSPPSS
RRPRPRRSRPPPP
SPSPSRRRPRSSS
RRPPPPPSRSPRS
PSPSPSPRPRSPP
SPRRSRSSPPSPR
SSSRSRPPPSPRS
PSSRRRSRRSPSS
PPRPRPSRSRSSP
PRRSSPSPPPRSR
PPPRPPPRSSRPS
PPSSSSPRPRPRR
SPPRRRPPRRPPS
SSPSPSRPSSPPR
SSPRRSSRPRPRR
PPRRPSRSSRPPR
RRPPRPSSRRSPP
PRPPPRPRPPPRP
PSPSSRRRPRSSP
RRRPPRPSRRPRR
PRRRRPPPSSPSR
RSRRSSSPPSRRP
RPPRSRSRSRPPR
SRPRPRSSSPPPP
PPRSPSRPPRPRR
PRPRSSPSSRPRP
SRSSRSPPPSPSS
RSRRRSPSRPSRP
RSSPRSPRSRSRR
RRSPPSPPRPRPS
PRSPPSRSSRSRS
SSPSSPRRSSSRS
SSRRRPSPRPPPR
SPSPSRSRSRPPR
SRRSRSRRPSRPR
SPSSSSSRPSPSS
SRSPPRRSSRSRS
PSRPPRSSPPRPR
SRRRPPPRSSPRS
RRSSRSRRPPRSS
SSSPSPSSRSRSR
SRSSPRRPRRSSR
PPSRSRRRPPSSS
SSRRPPPRRPRPR
RPPPSSPRSSPSR
RRSPSPRSRRPSR
