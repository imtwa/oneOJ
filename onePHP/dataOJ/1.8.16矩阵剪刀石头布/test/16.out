PRSPRSPSRSSRS
SRPRSPSRPSPPR
PRRSPPSRSRSSP
SSPRRRRPSPRRS
RPRPRPSRRRSPR
SPRSPRSRPSPPR
SRPSPPSRPPRSS
PSPRSPRPSRRPR
RPPSSPPRRPRRS
RPRRRPSPSSPRS
SSRPSPRSSRRSP
SPSPPRSPPPRSP
RPSPRRPRRSPRS
RPRSRSRRRPPSP
PSPPPRRPRRRPP
SRRPSPPSSRSRS
PPSSRPSRRPRSP
RRPPRSRPRSPRS
RPPPSPPSRSRSP
PSSRSRRSPRSPR
SRPRSSSSRSSPR
PSPRSRPRRPRSP
RPRPRRRSSRRPS
SRPSSPRRPSPPR
SRSRPRSPSPPSR
RPSRPSPRRSPPP
SRPRRPRRPRRRS
RPSPPSPPSPRRP
PSRSPRSSRPSSR
RPPPPSSPRRPPR
SPSRRPRRSSPPS
SPPSPPSSPRSRP
RPRPRRSPPSRPR
RSPRPSSPRPPSP
PPPPRRRPPSPRS
SRSSPPRSSRSPP
PSPRSSPPRPPRR
PSRRSRPRPSPRS
PSRPRRSSRRSSP
RPPSPPSRRPRSP
SRRRRSPPSSPRR
SRPPSPRRSRPSS
RPSRSPRRPRRPR
SRPRRSSRRSRPS
SRPSPSRPSSPRS
PSRSRSSRPRRSP
RPSSRPPSPSSPR
SRPPSRRPRPRSP
SSRPSSSPRPSPP
RPPSRPPRSRSRS
SSRPSPPSSRPRP
PPSRPPPPPSPSR
RRPSRSSPPSRSS
PSSRSRRRSRSPP
PSRPSPPRSPRSP
RRPRRPSRSPRSS
PPSPRRRPRRPRR
SRRPPRPSPRSSP
RRPSSPSRRSRSR
PRRRRSPPRSPRS
SPRPPPPSPRSPP
RSPSSSRRRSPRS
PSSRRSPPRSPPR
SRPPPSRSPPRSP
SSRSRPRPRRSPR
RPRSSRRPSSRSP
SPSPPSSRPSPRR
SRSPRRSSSRSPR
PSSRRPRRSPRSP
RPSRPSSPRSPRS
PSPSRRPPSSPRS
SRSPSSRRPRRSP
RPRSPPPSPRSPR
SSRSSRRPRRPRR
RRPRPRSRSSPSP
PSSPRSPSPRSSR
RPRSPRSPRRPPS
